src/main/java/com/google/sps/servlets/ElectionServlet.java

```diff
@@ -14,29 +14,19 @@
 
 package com.google.sps.servlets;
 
```

Comment:
Quick note about this file: I didn't change any code, the file just "rewrote" because the Java formatter fixed the indentation